aws-proofs/vm-steps.sh

```diff
@@ -107,7 +107,10 @@ fi
 # bundle up logs for artifact upload outside the VM
 # xz compression does help even if there are many .gz files in this set
 cd ~/.isabelle/heaps
-tar -Jcf ~/logs.tar.xz */log/*
+# usually one of the two globs below will not exist
+shopt -s nullglob
+# do not fail the test if log collection fails
+tar -Jcf ~/logs.tar.xz */log/* */*/log/* || touch ~/logs.tar.xz
```

Comment:
Do we care that an empty file is not valid input to `xz -d`? Probably not.

Reply:
you mean for uncompressing? It'll fail, but that is guarded as well on the other end.